### DISCUSSION TITLE
quickstart is broken on svelteKit?

### DISCUSSION THREAD

**Discussion by osbyrne** [material-components/material-web#5341] [Q&A] [answered]
Hi, I'm trying to use this wonderful UI kit, but I only get 
<img width="233" alt="image" src="https://github.com/material-components/material-web/assets/68084397/78c7453f-be8d-4596-bae3-658f3c6e784e">

steps to reproduce: 
> windows 11
> npm
> new sveltekit project (TS or JSdoc, I tried both)
> follow steps from https://material-web.dev/about/quick-start/ that is, npm i material UI, add imports in index.js, ..
> no UI on browser

I have no idea how to troubleshoot this issue, the console is empty, project runs with no errors, I installed the latest versions...
pls help!

**Comment by asyncliz** [comment]
Do you have a [build step](https://material-web.dev/about/quick-start/#building) that bundles the imports in `index.js`? If no, check out the building section in our quick start and use a bundler like [rollup](https://rollupjs.org/) or [webpack](https://webpack.js.org/).

If you do have a bundler build, can you post a link to an online reproduction with something like https://stackblitz.com/ for your project setup?

**Reply by osbyrne** [reply]
thanks for the recommendation
I installed rollup as shown in [get-started](https://github.com/material-components/material-web/blob/main/docs/quick-start.md)
still: 
stackblitz [REPL](https://stackblitz.com/~/github.com/osbyrne/testing-m3)
<img width="960" alt="image" src="https://github.com/material-components/material-web/assets/68084397/88699c59-cb8b-4463-838f-6bb51ab5662e">

**Reply by osbyrne** [reply, marked as answer]
ok I looked further and I see what I did wrong, my bad

I hadn't gotten to the last line to replace index.js by bundle.js
<script type="module" src="./index.js"></script>
becomes
<script src="./bundle.js"></script>
